ui/app/AppLayouts/Wallet/popups/SignTransactionModalBase.qml

```diff
@@ -66,12 +68,11 @@ StatusDialog {
     width: 480
     padding: 0
 
-    function formatBigNumber(number: string, decimals = -1) {
+    function formatBigNumber(number: string, symbol: string, options = null) {
```

Comment:
Is it worth moving to `utils` module? Should we stick to Qt5.15 js function definitions vs Qt6:
https://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html#method-attributes
vs
https://doc.qt.io/qt-6/qtqml-syntax-objectattributes.html#method-attributes

Reply:
Yeah, maybe later

Reply:
As for the signatures, we should stay with Qt5 but in the end it doesn't matter much atm; it's just a syntactic sugar, not enforced